tests/Issue.py

```diff
@@ -263,6 +264,9 @@ def testCreateReaction(self):
         self.assertEqual(reaction.id, 16917472)
         self.assertEqual(reaction.content, "hooray")
 
+    def testDeleteReaction(self):
+        self.issue.delete_reaction(85740167)
```

Comment:
With the above change, this can now be `self.assertTrue(self.issue.delete_reaction(85740167))`

Reply:
Code was updated accordingly. Resolving this conversation.